test/resources/services_no_umm_s.yml

```diff
@@ -0,0 +1,955 @@
+# Order for each CMR endpoint in this file will reflect precedence of the service when
```

Comment:
Can you make both of these test files more targeted to only a couple of services and make the names for the services match what we're trying to test? Something like `Missing UMM-S` instead of gesdisc/giovanni. 

So maybe one bad service and one good one in each file and only one CMR environment.

Reply:
I thought about doing this, but decided not to because I wanted to demonstrate that we can pick out the wrong service among a list of good ones with realistic configurations. 

I can see test files with customized service name and less configuration provides more clarity though. So I am OK (not thrilled) with your approach. I will make the change.